test/wpt/status/dom/events.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Event-dispatch-listener-order.window.js": { "fail": { "unexpected": [ "document is not defined" ] } },
+  "AddEventListenerOptions-passive.any.js": {
+    "fail": {
+      "unexpected": [
+        "assert_equals: Incorrect defaultPrevented for options: {\"passive\":true} expected false but got true",
+        "assert_equals: Incorrect defaultPrevented for options: undefined expected true but got false"
+      ]
+    }
+  },
+  "AddEventListenerOptions-signal.any.js": {
+    "fail": {
+      "unexpected": [
+        "assert_throws_js: function \"() => { et.addEventListener(\"foo\", () => {}, { signal: null }); }\" did not throw",
```

Comment:
I created PR to pass the test. However, it will occur a breaking change. So it won't be fixed.
https://github.com/nodejs/node/pull/43153